cheng-k/2025-Summer.md

```diff
@@ -6,32 +6,83 @@
 
 #### 🌹
 
-* 
+* I now have a good understanding about BIDS, goals it strives to address, and ideas it stands for. I am looking forward to continuing to work on these initiatives!
```

Comment:
```suggestion
* I now have a good understanding about BIDS, goals it strives to address, and ideas it stands for.
  I am looking forward to continuing to work on these initiatives!
```

Just a little edit to put each sentence on a new line. The two spaces at the start of the new line keep it inside of the bullet point formatting 😄